diff --git a/dist/index.js b/dist/index.js
index 212c4192672929de225cf58ba8bdcd99ef84f159..2d01c38931f57ae5f2906ee4ce62069723569eb1 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -22,7 +22,7 @@ function isInlineCode(element, parent, bypass = false) {
   return element.tagName === "code" && isElement(parent) && parent.tagName !== "pre" || element.tagName === "inlineCode";
 }
 function isBlockCode(element) {
-  return element.tagName === "pre" && Array.isArray(element.children) && element.children.length === 1 && isElement(element.children[0]) && element.children[0].tagName === "code";
+  return false;
 }
 function getInlineCodeLang(meta, defaultFallbackLang) {
   const placeholder = "\0";
